bin/prepare-vercel.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-PYTON_VERSION=3.12.1
+PYTHON_VERSION=3.12.1
```

Comment:
By the way, does Renovate already know how to update this version for us on this repository?

Maybe we need to change the Renovate configuration in this repository, or in the [`renovatebot/.github` repository](https://github.com/renovatebot/.github)?

Reply:
I'm not sure it does. @viceice ?

Reply:
can't updated automatically, because that binary isn't build automatically yet.

I've build it manually and uploaded to containerbase python-prebuild.

Reply:
Here's a new issue to track updating the python version variable:

- https://github.com/renovatebot/renovatebot.github.io/issues/389

I'm resolving this conversation, so we can merge the typo fix. 😉